{"name": "Попытка принять заказ с указанием неверного id курьера", "status": "passed", "start": 1706633041569, "stop": 1706633042171, "uuid": "b9f02d62-dbcb-4a85-bb30-b809a6ccca3b", "historyId": "e0236b6b2ccea89a76c9220d1b02f651", "testCaseId": "e0236b6b2ccea89a76c9220d1b02f651", "fullName": "tests.test_take_order.TestTakeOrder#test_take_order_with_incorrect_id_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "MBP-Ilya.cn.ru_novotelecom.ru"}, {"name": "thread", "value": "16501-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_take_order"}]}